CONTRIBUTING.md

```diff
@@ -26,64 +26,45 @@ ECS is an open source project and we love to receive contributions from our comm
 shared its intention to achieve convergence of ECS and OTel Semantic Conventions into a single standard maintained
 by OpenTelemetry.
 
-The stated plan has been to keep ECS in a frozen state during the transition. However, it is also apparent that these
-things take time. It takes time for the OTel community to adopt donated fields, and it will take time for development
-teams to build OTel native constructs in the Elastic stack. In the meantime, ECS users need to be able to develop
-features for Elastic that rely on continued contributions to the schemas that drive our technology.
-
-For these reasons, we need a process and guidelines for contributing to these data schemas during this period that
-allows us to avoid breaking changes.
 
-### ECS releases during the donation to OpenTelemetry
-Historically, ECS has shipped a new version with every minor release of the Elastic stack. While the schema is
-effectively frozen during the Otel donation period, this approach has been halted.
+#### How to contribute during OTel donation of ECS
 
-Moving forward, we will release ECS at the team's discretion as new material changes to the schema are adopted.
+During the contribution of ECS to OpenTelemetry, changes to the ECS schema can be made, but they should take into consideration how the changes
+can eventually be added to OpenTelemetry.
 
-While the decision to release will be discretionary, any release will still coincide with a minor Elastic stack release;
-however, not every minor version will warrant a new release of the stack.
+The [ECS RFC process](rfcs/PROCESS.md) will apply to larger, material changes to the schema, such as changes that add new use cases, top-level fieldsets, or could
+be considered controversial. Bug fixes or minor field addition changes can be made directly to the ECS project and submitted as pull requests.
 
-### How to contribute during OTel donation of ECS
+When writing the ECS RFC and designing fields, consider how it will be possible to contribute the field to semantic conventions in the future.
+Consider naming conventions, field definitions, mapping types, and conflicts with existing or proposed semantic conventions.
 
-Bug fixes or minor field addition changes can be made directly to the ECS project and submitted as pull requests.
+The same changes should be proposed to be added to [OpenTelemetry semantic conventions](https://github.com/open-telemetry/semantic-conventions)
+before, or in parallel with, adding to the ECS schema.
 
-Significant changes that add new use cases, top-level fieldsets, or could be considered controversial are
-considered material. The general rule for contributing new material changes to schemas during the transition period is
+The ECS RFC process should not advance past stage 2, moving fields from beta to GA, until the fields are accepted into OpenTelemetry.
```

Comment:
@mjwolf I know this might have been discussed offline, apologies for not keeping up:
- how will this affect rules development?
- why is general availability (which is a quality metric) tied into OTEL SemConv? I think we are trying to prevent breaking changes between ECS and OTEL but this will happen. Maybe we should consider an alternative solution?

Reply:
This wasn't discussed before. I thought it made sense to keep some way to allow breaking changes that might be needed to be compatible with otel, and keeping fields in beta could do it since beta fields are "subject to change".

You make a good point about how this would affect rules and other things outside the schema, and keeping fields in beta could block some usage.

I've taken this line out of the guidance. I think we can consider promoting fields to GA on a case-by-case basis, depending on how they're used and if there's any likely conflicts with otel.

Reply:
I agree with case by case situation.